jobs/env/ci-kubernetes-e2e-gce-large-correctness.env

```diff
@@ -1,5 +0,0 @@
-### cluster-env
-
-# TODO: Figure if we need to increase QPS for master components.
-# Increase service_controller's parallelism of processing service update
-CONTROLLER_MANAGER_TEST_ARGS=--concurrent-service-syncs=5
```

Comment:
Given that we don't bump it in 5k-node tests, I doubt we need to do this here.

Reply:
This wasn't a random change iirc. We intentionally changed this hoping that it helps with speeding up job when multiple LB-creating e2e tests run in parallel. But I think it didn't help much (and we eventually skipped those tests).

Reply:
I agree it wasn't a random change. But given that with 5k we don't need it, we also don't need it with 2k (assuming we are running the same set of tests which we do IIUC).

Reply:
Agreed.

That said I thought we were running lb tests on 2k (if not, we probably should try?)